cypress/support/awx-commands.ts

```diff
@@ -924,3 +925,24 @@ Cypress.Commands.add('waitForTemplateStatus', (jobID: string) => {
       }
     });
 });
+
+Cypress.Commands.add('waitForJobToProcessEvents', (jobID: string) => {
+  const waitForJobToFinishProcessingEvents = (maxLoops: number) => {
+    if (maxLoops == 0) {
+      cy.log('Max loops reached while waiting for processing events.');
+      return;
+    }
+    cy.wait(500);
+
+    cy.requestGet<Job>(`api/v2/jobs/${jobID}/`).then((job) => {
+      if (job.event_processing_finished !== true) {
+        cy.log(`EVENT PROCESSING = ${job.event_processing_finished}`);
+        cy.log(`MAX LOOPS = ${maxLoops}`);
+        waitForJobToFinishProcessingEvents(maxLoops - 1);
+      } else {
+        cy.log('Event processed');
+      }
+    });
+  };
+  waitForJobToFinishProcessingEvents(80);
```

Comment:
It would be nice to have a comment here explaining the reasoning behind the numbers chosen for wait and maxLoops. (i.e processing events takes ~30s so we chose 80 * 500ms as the upper limit)